README.md

```diff
@@ -42,7 +42,7 @@ In your pull requests, add a line to the end of the pull request description loo
 Or if you need a more precise, timezone-safe setting, you can use an ISO 8601 date string
 
 ```
-/schedule 2019-12-31T00:00:00.000Z
+/schedule 2019-12-31T00:00:00.000
```

Comment:
let's remove `.000` from readme. 
milliseconds accuracy is meaningless if github actions cron jobs only support min 5 min intervals on a 'best effort' basis

Reply:
Done